packages/synapse-react-client/src/components/SynapseSearchPageResults/SynapseSearchResultsCard.tsx

```diff
@@ -74,9 +153,15 @@ export function SynapseSearchResultsCard(props: SynapseSearchResultsCardProps) {
           }}
         >
           <FavoriteButton entityId={props.entityId} />
-          <Button variant="outlined" startIcon={<DownloadIcon />}>
-            Download
-          </Button>
+          <DropdownMenu
+            items={downloadMenuItems}
+            dropdownButtonText="Download"
+            buttonTooltip="Download options for this entity"
+            buttonProps={{
+              variant: 'outlined',
+              startIcon: <DownloadIcon />,
+            }}
+          />
```

Comment:
You may want to consider making this a reusable `EntityDownloadButton` (or some name to that effect) so the download logic is all captured there and decoupled from search. I could see us putting this in a lot of other places!